osc-control/src/main/java/org/osc/core/server/control/ServerControl.java

```diff
@@ -133,7 +134,7 @@ public boolean isRunning() {
                 throw new ParseException("no option specified");
             }
         } catch (ParseException e) {
-            log.info("Error parsing command line arguments", e);
+            getLogger(ServerControl.class).info("Error parsing command line arguments", e);
```

Comment:
Other places are using getLogger(this), but here you are using .class. Intentional?